bindings/go/osvschema/vulnerability.go

```diff
@@ -0,0 +1,170 @@
+package osvschema
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Package identifies the affected code library or command provided by the
```

Comment:
Standardize on the verb for consistency

```suggestion
// Package describes the affected code library or command provided by the
```